test/libsolidity/semanticTests/storage/mappings_array_pop_delete.sol

```diff
@@ -1,36 +0,0 @@
-contract C {
-	mapping (uint => uint)[] a;
-
-	function n1(uint key, uint value) public {
-		a.push();
-		a[a.length - 1][key] = value;
-	}
-
-	function n2() public {
-		a.push();
-	}
-
-	function map(uint key) public view returns (uint) {
-		return a[a.length - 1][key];
-	}
-
-	function p() public {
-		a.pop();
-	}
-
-	function d() public returns (uint) {
```

Comment:
Can we salvage this test by removing just this function? Similarly for the other test.

Reply:
Well, given that the whole point of the test is to test the deleting and popping (see the name) I didn't see much use in keeping it as both is not possible anymore now..